draft-ietf-keytrans-protocol.md

```diff
@@ -1826,15 +1867,106 @@ this document
+
+- Value: The numeric value of the cipher suite.
+- Name: The name of the cipher suite.
+- Recommended: Whether support for this cipher suite is recommended by the IETF.
+  Valid values are "Y", "N", and "D", as described below. The default value of
+  the "Recommended" column is "N". Setting the Recommended item to "Y" or "D",
+  or changing an item whose current value is "Y" or "D", requires Standards
+  Action {{!RFC8126}}.
+  - Y: Indicates that the IETF has consensus that the item is RECOMMENDED. This
+    only means that the associated mechanism is fit for the purpose for which it
+    was defined. Careful reading of the documentation for the mechanism is
+    necessary to understand the applicability of that mechanism. The IETF could
+    recommend mechanisms that have limited applicability, but it will provide
+    applicability statements that describe any limitations of the mechanism or
+    necessary constraints on its use.
+  - N: Indicates that the item has not been evaluated by the IETF and that the
+    IETF has made no statement about the suitability of the associated
+    mechanism. This does not necessarily mean that the mechanism is flawed, only
+    that no consensus exists. The IETF might have consensus to leave an item
+    marked as "N" on the basis of it having limited applicability or usage
+    constraints.
+  - D: Indicates that the item is discouraged and SHOULD NOT or MUST NOT be
+    used. This marking could be used to identify mechanisms that might result in
+    problems if they are used, such as a weak cryptographic algorithm or a
+    mechanism that might cause interoperability problems in deployment.
+- Reference: The document where this cipher suite is defined.
+
+Initial contents:
+
+| Value  | Name                                 | R | Ref      |
+|:-------|:-------------------------------------|:--|:---------|
+| 0x0000 | RESERVED                             | - | RFC XXXX |
+| 0x0001 | KT_128_SHA256_P256                   | Y | RFC XXXX |
+| 0x0002 | KT_128_SHA256_Ed25519                | Y | RFC XXXX |
+
+For all cipher suites this document defines, the following primitives and
+parameters are the same:
+
+- The hash algorithm is SHA-256, as defined in {{SHS}}.
+- `Nc`: 16
+- `Kc`: The hex-decoded value: `d821f8790d97709796b4d7903357c3f5`
+
+The signature algorithm and VRF algorithm for each cipher suite is as follows:
+
+| Name                  | Signature              | VRF Algorithm                     |
+|:----------------------|:-----------------------|:----------------------------------|
+| KT_128_SHA256_P256    | ecdsa_secp256r1_sha256 | ECVRF-P256-SHA256-TAI             |
+| KT_128_SHA256_Ed25519 | ed25519                | ECVRF-EDWARDS25519-SHA512-TAI[32] |
+
+The VRF algorithms are specified in {{RFC9381}}. For `KT_128_SHA256_Ed25519`,
+the final hash output of `ECVRF-EDWARDS25519-SHA512-TAI` is truncated to be 32
+bytes.
 
 ## KT Designated Expert Pool {#de}
```

Comment:
Oy. This seems like quite the process being standardized here. Has this already been worked towards? This must be carefully discussed before we proceed. I would like to learn more about the requirements for having a "designated experts" group etc. Why not just leave it up to the WG?

Reply:
I'm pretty sure it is just a mailing list to approve allocating codepoints. TLS, MLS, etc have them. The keytrans wg may not always exist to make such decisions